utils/src/ParticleIDUtils.cc

```diff
@@ -11,6 +13,22 @@
 
 namespace edm4hep::utils {
 
+namespace {
+  int32_t getAlgoID(const std::string& name) {
+    int32_t ID = 0;
+    MurmurHash3_x86_32(name.c_str(), name.size(), 0, &ID);
```

Comment:
If we only use this hash implementation, we could drop the other unused implementations.